packages/eslint-mdx/src/traverse.ts

```diff
@@ -56,28 +56,37 @@ export class Traverse {
           // prettier-ignore
           /* istanbul ignore next */
           else if (
```

Comment:
The most important code change is the following lines.

https://github.com/mdx-js/eslint-mdx/pull/273/files#diff-b32281cf6a9e6127915ce948b00b48377679fa085e7d45d451fe7769ae9ee351R58-R89